pyscriptjs/src/components/pyscript.ts

```diff
@@ -187,5 +194,22 @@ async function mountElements() {
     }
     await pyodide.runPythonAsync(source);
 }
+
+async function initNamespaces() {
+    console.log('Copying global namespace to separate namespaces if necessary');
+    const pyodide = await pyodideReadyPromise;
+    const matches: NodeListOf<HTMLElement> = document.querySelectorAll('[pys-namespace]');
```

Comment:
if we change the attribute name, this will need to change to query by tag and attribute

Reply:
Agreed, see thoughts above.